packages/nodes/table/src/queries/getTableRowIndex.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  findNodePath,
+  TElement,
+  TReactEditor,
+  Value,
+} from '@udecode/plate-common';
+
+/**
+ * Get table row index of a cell node.
+ */
+export const getTableRowIndex = <V extends Value>(
+  editor: TReactEditor<V>,
+  cellNode: TElement
+) => {
+  const path = findNodePath(editor, cellNode);
+  if (!path) return 0;
+  return path[path.length - 2];
+};
```

Comment:
can be unit tested

Reply:
I tried addings tests for `getTableColumnIndex` prior to writing this function, and I ran into a weird issue where `findNodePath` seemed to always return undefined. Presumably it's because Slate's weak maps aren't getting set. Any tips for working around that?